src/client.rs

```diff
@@ -363,30 +364,43 @@ mod tests {
                 .build(),
             valid_latest_change_response,
             r#"{
-            "metadata": {},
-            "changes": [],
-            "timestamp": 0
-        }"#,
+                "metadata": {},
+                "changes": [],
+                "timestamp": 0
+            }"#,
             Err(ClientError::Error {
                 name: "invalid signature error".to_owned(),
             }),
         );
 
-        test_get(&mock_server,Client::builder()
-            .server_url(&mock_server_address)
-            .collection_name("url-classifier-skip-urls")
-            .verifier(Box::new(VerifierWithNoError {}))
-            .build(), &format!(
-            "{}",
+        test_get(
+            &mock_server,
+            Client::builder()
+                .server_url(&mock_server_address)
+                .collection_name("url-classifier-skip-urls")
+                .verifier(Box::new(VerifierWithNoError {}))
+                .build(),
+            &format!(
+                "{}",
+                r#"{
+                    "metadata": {},
+                    "changes": [],
+                    "timestamp": 0
+                }"#
+            ),
```

Comment:
I realize that this is pre-existing, but why `format!("{}", r#"..."#)` instead of just `r#"..."`?

```suggestion
            r#"{
                "metadata": {},
                "changes": [],
                "timestamp": 0
            }"#
```